functional-tests/src/test/groovy/specs/app/LoginSpec.groovy

```diff
@@ -42,4 +31,101 @@ class LoginSpec extends FixtureSpec {
       ClickLink           || AssertPage
       "ForgotPasswordBtn" || ForgotPasswordPage
   }
+
+  def "Navigate Page from: LoginPage, log in as Admin, Assert Page: HomePage"() {
+
+    given: "I start on the LoginPage, am not logged in, and have an account"
+      fixture_files =['admin_user_fixture']
+      fixture(FixtureSpec.INSERT)
+      to LoginPage
+    when: "and I log in with valid credentials"
```

Comment:
Nit.: *when* statements are generally in the form "I...somethingsomething". Not sure if it is for compliace with Gherkin or what, but I would stick with that (same for next couple statements).